examples/util/base.js

```diff
@@ -207,10 +197,10 @@ const densenet_onnx = {
   },
   intro: 'Dense Convolutional Network (DenseNet) connects each layer to every other layer in a feed-forward fashion. DenseNets have several compelling advantages: they alleviate the vanishing-gradient problem, strengthen feature propagation, encourage feature reuse, and substantially reduce the number of parameters. ',
   paperUrl: 'https://arxiv.org/abs/1608.06993'
-};
+}];
 
-const ssd_mobilenetv1_tflite = {
-  modelName: 'SSD MobileNet v1(TFlite)',
+const objectDetectionModels = [{
+  modelName: 'ssd_mobilenet_v1_tflite',
```

Comment:
@ibelem This modification will let model name of Object Detection example show as 'ssd_mobilenet_v1_tflite' not previous 'SSD Mobilenet v1(TFlite)' on UI.

Reply:
@BruceDai The new UI in #558 will have no such issue. But I am OK to add new fields as below, please confirm if any conflict base on following code.


```
const imageClassificationModels = [{
  modelName: 'Mobilenet v1 (TFLite)',
  modelFormatName: 'mobilenet_v1_tflite',

```

```
const objectDetectionModels = [{
  modelName: 'SSD MobileNet v1 (TFLite)',
  modelFormatName: 'ssd_mobilenet_v1_tflite',
```

I will submit new commit if no issues for you.

Reply:
@ibelem No conflict,  I think it would be OK to do so.